app/scripts/content/QtumProvider.ts

```
@@ -0,0 +1,406 @@
+        // this.wrapResults.delete(payload.id);
+        console.log("unhandled", payload);
+        payload.jsonrpc = "2.0";
+        // this.rpc
+        //   .call(payload)
+        //   .then((response: any) => {
+        //     if (this.isDebug) {
+        //     }
+        //     wrapResult ? resolve(response) : resolve(response.result);
+        //   })
+        //   .catch(reject);
+        const proxyResult = await this.proxyRPCCall(payload);
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)